packages/ndla-ui/src/ProgrammeCard/ProgrammeCard.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import SafeLink from '@ndla/safelink';
+import { css } from '@emotion/react';
+import { spacing, colors, misc, breakpoints, mq } from '@ndla/core';
+import { HeadingLevel } from '../types';
+
+interface Image {
+  src: string;
+  alt: string;
+}
+export interface Programme {
+  id: string;
+  programmeTitle: {
+    title: string;
+    language: string;
+  };
+  headingLevel?: HeadingLevel;
+  programmeIMGDesk: Image;
+  programmeIMGMob: Image;
+  url: string;
+}
+
+const StyledCardContainer = styled(SafeLink)`
+  ${mq.range({ from: breakpoints.tablet })} {
+    height: 350px;
+    width: 250px;
+  }
+  max-width: 350px;
+  height: 195px;
+  display: flex;
+  flex-direction: column;
+  background-color: ${colors.background.default};
+  border-radius: ${misc.borderRadius};
+  margin-bottom: ${spacing.nsmall};
```

Comment:
Det er en veldig grei tommelfingerregel å revurdere all bruk av margin på en komponent. Da fjerner man plutselig mye av mulighetene komponent-komposisjon gir deg. Det er nesten alltid container-wrapperen som står ansvarlig for margin mellom komponenter.

Reply:
Fjernet.